lib/KBreadcrumbs.vue

```diff
@@ -4,7 +4,7 @@
     v-show="showSingleItem || crumbs.length > 1"
     :class="{ 'breadcrumbs-collapsed': collapsedCrumbs.length }"
   >
-    <nav class="breadcrumbs">
+    <nav class="breadcrumbs" v-bind="$attrs" :aria-label="$attrs.ariaLabel">
```

Comment:
I think we should also add `inheritAttrs: false` to the component as well? https://v2.vuejs.org/v2/api/#inheritAttrs

Reply:
Like we have in the `TextTruncatorCss` component? https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/TextTruncatorCss.vue#L42

Reply:
Yes, it's possible it's not actually required, but this avoids the `$attrs` being erroneously applied to the containing div.

Reply:
@rtibbles added